wiki/Contests/VMC/en.md

```diff
@@ -0,0 +1,14 @@
+# Vocaloid Mapping Contest
+
+![VMC banner](https://cdn.discordapp.com/attachments/1092234806479298680/1096272980310495272/banner_e.png)
+
+Index page for all of the Vocaloid Mapping Contest series.
+
+- [Vocaloid Mapping Contest](1)
```

Comment:
the previous iteration is listed like this, even if it doesn't have an article yet. i've copied the title from the forum post you linked

```suggestion
- Vocaloid Mapping Competition
- [Vocaloid Mapping Contest](1)
```